user/templates/windows/qubes-windows-tools.rst

```diff
@@ -143,12 +160,25 @@ If script execution is blocked, it must be allowed using the following PowerShel
 
 	PS C:\> Set-ExecutionPolicy Unrestricted
 
+**Warning:** It is strongly suggested to enable autologon for any Windows HVMs that will have Qubes Tools installed. To do so, in Windows 7, run the ``netplwiz`` command from the :kbd:`Win-R` "execute" prompt, and uncheck the *Users must enter a user name and password to use this computer* option. In Windows 10 and 11, autologon can be configured later on during Qubes Windows Tools installation.
```

Comment:
"strongly suggested" is weird, "encouraged" would be better. But why is it strongly suggested to enable autologon?